test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -1828,7 +1828,8 @@ def convert_to_dictlist(self, l, attribs=None, mergelines=True, id=None):
                     _e = len(lines) - 1
                     lines[_e] = lines[_e].strip('\r\n\t') + \
                         l[i].strip('\r\n\t')
-                elif not l[i].startswith(' ') and l[i-count].startswith('\t'):
+                elif not l[i].startswith(' ') and l[i-count].startswith('\t') \
```

Comment:
we need to put `i > count` before `l[i-count].startswith('\t')`. So change this line as below:
```
elif (not l[i].startswith(' ') and i > count
        and l[i-count].startswith('\t')):
```